.env.example

```
@@ -0,0 +1,22 @@
+
+# 1. OpenAI Settings
+# GPT-4o 모델 호출 및 임베딩 생성을 위한 인증 키
+OPENAI_API_KEY="YOUR_OPENAI_API_KEY"
+
+# 2. Database Settings
+# 벡터 데이터베이스 ChromaDB의 데이터가 저장될 로컬 경로
+CHROMA_DB_PATH="./data/chroma"
+
+# 3. LangChain / LangGraph Settings (Optional)
+# 에이전트의 추론 과정을 모니터링하기 위한 트레이싱 활성화 여부
+LANGCHAIN_TRACING_V2="false"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)